pybloom/pybloom.py

```diff
@@ -282,7 +284,7 @@ def __getstate__(self):
 
     def __setstate__(self, d):
         self.__dict__.update(d)
-        self.make_hashes = make_hashfuncs(self.num_slices, self.bits_per_slice)
+        self.make_hashes, self.hashfn_name = make_hashfuncs(self.num_slices, self.bits_per_slice)
```

Comment:
Would it be useful to have a simple unit test to ensure that `make_hashfuncs` returns 2 params and what they are?